src/Layers/xrRenderPC_R2/r2_loader.cpp

```diff
@@ -264,13 +258,11 @@ void CRender::LoadBuffers(CStreamReader* base_fs, bool alternative)
             Msg("* [Loading IB] %d indices, %d Kb", iCount, (iCount * 2) / 1024);
 
             // Create and fill
-            BYTE* pData = 0;
-            R_CHK(HW.pDevice->CreateIndexBuffer(iCount * 2, dwUsage, D3DFMT_INDEX16, D3DPOOL_MANAGED, &_IB[i], 0));
-            HW.stats_manager.increment_stats_ib(_IB[i]);
-            R_CHK(_IB[i]->Lock(0, 0, (void**)&pData, 0));
+            _IB[i].Create(iCount * 2);
```

Comment:
Old code doesn't use `D3DUSAGE_SOFTWAREPROCESSING` for `dwUsage` in software mode, but `_IB[i].Create` has it.

Reply:
This odd for level geometry loading only. I believe this typo came from past times and the flag should be set. See R1 for reference.
Any reasons to treat this particular geometry buffers in different manner? Am I missing smth?

Reply:
> This odd for level geometry loading only. I believe this typo came from past times and the flag should be set. See R1 for reference.
Any reasons to treat this particular geometry buffers in different manner? Am I missing smth?

@SkyLoaderr, so, do you have something to say about it?

Reply:
I bet this is not a typo and was made intentional. But, however, R2 probably won't even launch with hardware features, so this should be safe.